VideoLocker/res/values/strings.xml

```diff
@@ -254,8 +254,10 @@
     <!-- Assessment -->
     <!-- Label default unit on assessment navigation bar-->
     <string name="assessment_next_unit">Next Unit</string>
-    <!-- Label showing prvious on assessment navigation bar-->
```

Comment:
Every item here needs a comment. These go to our translators so they can know some context about each string.